pkg/kwokctl/cmd/config/tidy/tidy.go

```diff
@@ -33,7 +33,7 @@ func NewCommand(ctx context.Context) *cobra.Command {
 	cmd := &cobra.Command{
 		Args:  cobra.NoArgs,
 		Use:   "tidy",
-		Short: "Tidy the default config file with --config",
+		Short: "Tidy the default config file",
```

Comment:
Here the `--config` actually merges the default configuration with the configuration it specifies into the default configuration

Reply:
Good point! Thanks, Shiming.